src/harness/unittests/reuseProgramStructure.ts

```diff
@@ -1013,30 +994,27 @@ namespace ts {
                     ]
                 }
             };
-            const app: FileOrFolder = {
-                path: "/src/packages/framework/app.ts",
-                content: 'import classc from "module1/lib/file1";\
-                          import classD from "module3/file3";\
-                          let x = new classc();\
-                          let y = new classD();'
-            };
-            const module1: FileOrFolder = {
-                path: "/src/packages/mail/data/module1/lib/file1.ts",
-                content: 'import classc from "module2/file2";export default classc;',
-            };
-            const module2: FileOrFolder = {
-                path: "/src/packages/mail/data/module1/lib/module2/file2.ts",
-                content: 'class classc { method2() { return "hello"; } }\nexport default classc',
-            };
-            const module3: FileOrFolder = {
-                path: "/src/packages/styles/module3/file3.ts",
-                content: "class classD { method() { return 10; } }\nexport default classD;"
-            };
-            const configFile: FileOrFolder = {
-                path: "/src/tsconfig.json",
-                content: JSON.stringify({ compilerOptions, include: ["packages/**/*.ts"] })
-            };
-            verifyProgramWithConfigFile(createTestSystem([app, module1, module2, module3, libFile, configFile]), configFile.path);
+            const configFile = "/src/tsconfig.json";
+            const fs = new vfs.FileSystem(/*ignoreCase*/ false, { cwd: "/", files: {
+                "/src/packages/framework/app.ts":
+                    `import classC from "module1/lib/file1";\n` +

```

Comment:
Lines like these make for a good argument for the `dedent` template tag function.

Reply:
I'd prefer to do another pass later, in a separate PR.